cli/command/formatter/disk_usage.go

```diff
@@ -65,7 +65,7 @@ reclaimable: {{.Reclaimable}}
 }
 
 func (ctx *DiskUsageContext) Write() (err error) {
-	if ctx.Verbose == false {
+	if !ctx.Verbose {
```

Comment:
Ya, this is one of the `gosimple` checks. It's a minor thing, but in a longer expression it's probably easier to read without the `== true/false`.